src/Dapr.Actors/Runtime/TimerInfo.cs

```diff
@@ -8,16 +8,15 @@ namespace Dapr.Actors.Runtime
     using System;
     using System.Globalization;
     using System.IO;
-    using System.Text;
     using System.Text.Json;
     using System.Text.Json.Serialization;
     using System.Threading;
-    using System.Threading.Tasks;
     using Dapr.Actors.Resources;
 
     /// <summary>
     /// Represents the details of the timer set on an Actor.
     /// </summary>
+    [Obsolete("This class is an implementation detail of the framework and will be made internal in a future release.")]
```

Comment:
Unfortunately this was public, despite there not really being anything useful to do with it.